### PR TITLE
Support Cake 0.38.0

### PR DESCRIPTION
**Pull request by bdukes**
`TFBuild` was renamed to `AzurePipelines` in Cake 0.38.  This PR updates the usages and also updates the naming within the `Cake.TFBuild.Module` to `Cake.AzurePipelines.Module`.

I _think_ this is the only thing that would prevent this from [working on 1.0.0](https://github.com/agc93/Cake.BuildSystems.Module/issues/24) also.

Let me know if there are any adjustments I can make for you.

Thanks!

### CODE REVIEW

**Comment by nils-a** [comment]
@bdukes are you in a position to rebase this onto the current develop?

File: azure-pipelines.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ trigger:
 
 variables:
   buildConfiguration: 'Release'
-  cakeVersion: 0.34.1
+  cakeVersion: 0.38.0
```
Comment (nils-a) [inline azure-pipelines.yml:7]: This should be 0.38.5, as this is the Cake version we're building with. (Not the Cake version we're building against...)

**Review by nils-a — CHANGES_REQUESTED** [review]
One minor change, the rest LGTM.

File: azure-pipelines.yml (hunk 1 above)
Comment (bdukes) [reply azure-pipelines.yml]: I've pushed this change (along with the rebase).  Thanks!

**Review by nils-a — APPROVED** [review]
LGTM

**Comment by nils-a** [comment]
@bdukes your changes have been merged, thanks for your contribution :+1:

**Comment by bdukes** [comment]
@nils-a Thanks! Any chance this change could be published sometime soon?

**Comment by nils-a** [comment]
@bdukes I'll try 😄 
I want to get the [milestone](https://github.com/cake-contrib/Cake.BuildSystems.Module/milestone/1) ready, then :shipit: 

(Only complex things in there are 28 and 29 - 28 has a PR and is being reviewed, 29 is prepared and only waiting for 28 to be merged.)

**Comment by bdukes** [comment]
:+1: thanks for the update

**Comment by nils-a** [comment]
@bdukes I released version 1.0.0 which supports Cake 0.38.0 (i.e. this PR) - will update to Cake 1.0.0 (and release a new version of Cake.Buildsystems.Module) soon.

**Comment by bdukes** [comment]
Thanks @nils-a! I saw the release and am already running a build with it.  It [appears to be working](https://dev.azure.com/dotnet/DNN/_build/results?buildId=45547&view=ms.vss-build-web.run-extensions-tab) on our project using Cake 1.0.0, are there features you would expect not to work?

**Comment by nils-a** [comment]
@bdukes No. I gave it a shakedown using the rc0003 when I tested your PR (nothing complex) and everything was fine.

**Pull request merged**